src/egl/drivers/dri2/egl_dri2.h

```diff
@@ -59,6 +59,7 @@
 #include <system/window.h>
 #include "platform_android_gralloc_drm.h"
 #include <hardware/gralloc1.h>
```

Comment:
@munish-b Feel free to merge the changes https://github.com/android-ia/external-mesa/commit/faa74e181e9291e950460c5d3d62c47343c14757 so we have 1 PR for review.